src/components/VmModals/VmYesNoModal.js

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes } from 'react'
```

Comment:
The class is already written to be non-Vm specific. Nice!

Please rename and move this class to be more general.

<YesNoModal /> is enough.